### ISSUE TITLE
FirstDeliveryDate returns NULL causing issues when creating slips/labels

### ISSUE THREAD

**Issue by thompalstra** [tig-nl/postnl-magento2#82]
## Please follow the guide below

- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like.
- We may ask some questions or ask you to provide addition information after you placed your request.

---

### Make sure you are using the *latest* version: https://tig.nl/postnl-magento-extensies/
Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running the latest version of the TIG PostNL Magento extension.

---

### What is the purpose of your *issue*?
- [ ] Feature request (request for a new functionality)
- [x] Bug report (encountered problems with the TIG PostNL Magento extension)
- [ ] Extension support request (request for adding support for a new extension)
- [ ] Other

### If the purpose of your issue is a *bug report*

Step 1: Submit order via frontend without a correct postcode/country combination
Step 2: Order is created normally
Step 3: Trying to create packing slips/labels will result in the following error: 

Expected result: Create packing slips

Actual result: Check CIFException in the detail section - 2034: Bezorgdatum, mag geen maandag zijn indien er geen zondagsorting is.

The error becomes an issue when an order has been created with incorrect data and the tig_postnl_order  delivery_date has been set to NULL. 

The issue has been traced back into the following model:
Service/Order/FirstDeliveryDate.php 99

```
    private function getDeliveryDate(Address $address)
    {
        try {
            $this->endpoint->setStoreId($this->quote->getStoreId());
            $this->endpoint->setParameters([
                'country'  => $address->getCountryId(),
                'postcode' => $address->getPostcode(),
            ]);
            
            $response = $this->endpoint->call();
        } catch (\Exception $exception) {
            return null; << value causes issues
        }
        
        if (is_object($response) && $response->DeliveryDate) {
            return $response->DeliveryDate;
        }
        
        return null; << value causes issues
    }
```

Current workaround: manually set database record delivery_date from NULL to an actual date.

**Comment by tig-rikjonkmans** [comment]
Hi @thompalstra ,

Thanks for reporting this issue, this will be solved in the upcoming release, which is currently in the testing stage.

**Comment by tig-rikjonkmans** [comment]
Just released version 1.7.0 which contains a fix for this problem. 

Please use the following command to update  `composer update tig/postnl-magento2 --with-dependencies`
